src/pages/minjiJeong/Main/Comment/Comment.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import CommentList from './CommentList';
+import './Comment.scss';
+
+let id = 4;
+
+class Comment extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      commentList: [],
+      content: '',
+      isLiked: false,
+    };
+  }
+
+  componentDidMount() {
+    this.setState({ commentList: this.props.commentList });
```

Comment:
위쪽 state가 되어야 하는 값 코멘트에서 안내드렸듯이 props로 받은 값은 state로 관리하시면 안됩니다!
commentList의 변화가 필요한 경우에는 props로 받은 commentList 또한 부모의 state일거니까 부모의 state를 변경하는식으로 적용해주세요

Reply:
생각해보니까 굳이 state값에 props를 안넣고 그냥 바로 this.props.commentList로 map을 돌려도 되는 것이었네요!
저 부분은 feed 데이터와 comment 데이터가 하나의 json 파일에 있을 때 적용한 코드였는데,
현재는 두 파일이 분리되어 있어서 
```suggestion
    this.setState({
      commentList: COMMENT_LIST[this.props.feedId],
    });
```
이런 식으로 받아오고 있습니다.
이 경우에도 state값에 feedId라는 props 값을 넣어주고 있는데 이런 방식으로 하면 안되는 것인가요??